lively.collab/comments/commentIndicator.js

```diff
@@ -70,6 +70,11 @@ export class CommentIndicator extends Label {
     $world.addMorph(this);
   }
 
+  copy (trueCopy) {
```

Comment:
When is copy also called (so that `trueCopy` is a necessary parameter)?

Reply:
So, `copy` returns true if the copy process is valid/successful, false otherwise?

Reply:
The return value here was confusing, we implemented a better solution to the problem that comes with the next push.

`trueCopy` is there because the `copy()` method is also called when a Morph gets dragged and at multiple strange times e.g., when a browser is opened. We are not sure why this is, but in these cases there is no actual copying taking place (for us an actual copy is e.g., pressing Strg-C). In order to not break all of these, we introduced the parameter that defaults to `false`.